app/v1/models/request/liability/LiabilityRawData.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.request.liability
+
+import v1.models.request.RawData
+
+case class LiabilityRawData(vrn: String,
+                            from: String,
+                            to: String) extends RawData
```

Comment:
Because 'from' and 'to' are mandatory query parameters they need to be made optional in the raw request. This is because if we make them mandatory in the route and they are not supplied then the vendor will be returned a default Play 404 and we won't be able to return our custom error for this scenario.
```suggestion
                            from: Option[String],
                            to: Option[String]) extends RawData
```

Reply:
By making them optional in the raw request and mandatory in the parsed request we can validate against their existence in the request parser and throw our 'INVALID_TO' or "INVALID_FROM' errors as required.